style.css

```diff
@@ -116,12 +162,44 @@ article h1 {
   margin: 0;
 }
 
+.about-me li {
+  margin: 0 10px 0 0;
+}
+
 .social-links ul {
   list-style: none;
   display: flex;
   padding-right: 10px;
 }
 
+.frame2 li {
+  margin-left: 30px;
+  font-family: 'Poppins' Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.tags li {
+  color: #3b4eff;
+  font: 500 12px;
+  padding: 0 2px 0 -21px;
```

Comment:
Kindly change the value of the `padding` here to `5px 10px` so your design will match the Figma design 😉 

 - Your design:
   
![Care](https://user-images.githubusercontent.com/95412943/212365237-ab1f009b-9e19-429d-bc8c-d55110dac8bd.PNG)

 - The Figma design:
   ![figma](https://user-images.githubusercontent.com/58642949/212356096-8b16dda3-0ff5-4187-92fc-299b41403588.png)